extra/man/alacritty.5.scd

```diff
@@ -308,122 +302,145 @@ Colors are specified using their hexadecimal values with a _#_ prefix: _#RRGGBB_
 	*blue* <string>
-		Default: _"#82b8c8"_
 	*magenta* <string>
-		Default: _"#c28cb8"_
 	*cyan* <string>
-		Default: _"#93d3c3"_
 	*white* <string>
-		Default: _"#f8f8f8"_
 
 *dim*
 
 	If the dim colors are not set, they will be calculated automatically based
 	on the _normal_ colors.
 
 	*black* <string>
-		Default: _"#0f0f0f"_
 	*red* <string>
-		Default: _"#712b2b"_
 	*green* <string>
-		Default: _"#5f6f3a"_
 	*yellow* <string>
-		Default: _"#a17e4d"_
 	*blue* <string>
-		Default: _"#456877"_
 	*magenta* <string>
-		Default: _"#704d68"_
 	*cyan* <string>
-		Default: _"#4d7770"_
 	*white* <string>
-		Default: _"#8e8e8e"_
 
 *indexed_colors* [{ index = <integer>, color = <string> },]
 
 	The indexed colors include all colors from 16 to 256.
 	When these are not set, they're filled with sensible defaults.
 
-	Default: _[]_
+	Example:
+
+		*[colors]*++
+indexed_colors = [++
+    { index = _16_, color = _"#ff00ff"_ },++
+]
 
 *transparent_background_colors* <boolean>
 
 	Whether or not _window.opacity_ applies to all cell backgrounds, or only to
 	the default background. When set to _true_ all cells will be transparent
 	regardless of their background color.
 
-	Default: _false_
-
 *draw_bold_text_with_bright_colors* <boolean>
 
 	When _true_, bold text is drawn using the bright color variants.
 
-	Default: _false_
+Default:
+
+	*[colors]*++
+draw_bold_text_with_bright_colors = _false_++
+transparent_background_colors     = _false_
+
+	line_indicator = { foreground = _"None"_, background = _"None"_ }++
+footer_bar     = { foreground = _"#181818"_, background = _"#d8d8d8"_ }++
+cursor         = { text = _"CellBackground"_, cursor = _"CellForeground"_ }++
+vi_mode_cursor = { text = _"CellBackground"_, cursor = _"CellForeground"_ }++
+selection      = { text = _"CellBackground"_, background = _"CellForeground"_ }
+
+	*[colors.primary]*++
+foreground        = _"#d8d8d8"_++
+background        = _"#181818"_++
+dim_foreground    = _"#828482"_++
+bright_foreground = _"None"_
+
+	*[colors.hints]*++
+start = { foreground = _"#181818"_, background = _"#f4bf75"_ }++
+end   = { foreground = _"#181818"_, background = _"#ac4242"_ }
+
+	*[colors.search]*++
+matches       = { foreground = _"#181818"_, background = _"#ac4242"_ }++
+focused_match = { foreground = _"#181818"_, background = _"#f4bf75"_ }
+
+	*[colors.normal]*++
+black   = _"#181818"_++
+red     = _"#ac4242"_++
+green   = _"#90a959"_++
+yellow  = _"#f4bf75"_++
+blue    = _"#6a9fb5"_++
+magenta = _"#aa759f"_++
+cyan    = _"#75b5aa"_++
+white   = _"#d8d8d8"_
+
+	*[colors.bright]*++
+black   = _"#6b6b6b"_++
+red     = _"#c55555"_++
+green   = _"#aac474"_++
+yellow  = _"#feca88"_++
+blue    = _"#82b8c8"_++
+magenta = _"#c28cb8"_++
+cyan    = _"#93d3c3"_++
+white   = _"#f8f8f8"_
+
+	*[colors.dim]*++
+black   = _"#0f0f0f"_++
+red     = _"#712b2b"_++
+green   = _"#5f6f3a"_++
+yellow  = _"#a17e4d"_++
+blue    = _"#456877"_++
+magenta = _"#704d68"_++
+cyan    = _"#4d7770"_++
+white   = _"#8e8e8e"_
```

Comment:
This seems excessive. Wouldn't it be better if each block was below its actual table?

Reply:
I just wanted to provide a full color scheme here.

Reply:
That's fair but it seems excessive.